core/state/statedb.go

```diff
@@ -1297,20 +1351,48 @@ func (s *StateDB) Commit(failPostCommitFunc func(), postCommitFuncs ...func() er
 	var diffLayer *types.DiffLayer
 	var verified chan struct{}
 	var snapUpdated chan struct{}
+	var snapCreated chan common.Hash
```

Comment:
Already have `snapUpdated`, no need to have another `snapCreated`

Reply:
yes